src/v/cluster/scheduling/leader_balancer.cc

```diff
@@ -226,8 +228,10 @@ ss::future<ss::stop_iteration> leader_balancer::balance() {
     if (!transfer) {
         vlog(
           clusterlog.info,
-          "No leadership balance improvements found with total delta {}",
-          error);
+          "No leadership balance improvements found with total delta {} muted "
+          "groups {}",
```

Comment:
I find this log message a bit hard to read :) Maybe a better one is `... total delta {}, number of muted groups: {}`

Reply:
fixed